vss-extension.json

```diff
@@ -0,0 +1,36 @@
+{
+    "manifestVersion": 1,
+    "id": "notation2",
```

Comment:
https://marketplace.visualstudio.com/items?itemName=adrienld.notation
There is another tool also named as `notation` in the Marketplace, so it will report an error if I use `"id": "notation"`. The `vss-extension.json` file is for marketplace. Finally, we will push the task to azure-pipelines-tasks repo, so we don't need the file at that moment.

Reply:
How about `notarynotation`?

Reply:
`notarynotation` works.
The ID is for private share through Marketplace. We dont't need it when we push to azure-pipelines-tasks repo.